src/main/resources/META-INF/rewrite/was-to-liberty.yml

```diff
@@ -52,6 +52,11 @@ recipeList:
       performStaticCall: true
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.liberty.WebSphereUnavailableSSOCookieMethod
+recipeList:
+  - org.openrewrite.java.RevokeSSOCookiesToLogout
```

Comment:
Could it make sense to rename the class to `WebSphereUnavailableSSOCookieMethod` and do away with the Yaml recipe here? That has the same effect, without this intermediate that lacks a `displayName` and `description`.

Reply:
We  are still using the RevokeSSOCookiesToLogout class which has the displayName and description right? Is that a problem?

Reply:
I think Tim is saying to `RevokeSSOCookiesToLogout` to `WebSphereUnavailableSSOCookieMethod`. That way we would no longer need this entry in the Yaml file.

Reply:
Perhaps this change does a better job of getting across what I had intended: https://github.com/openrewrite/rewrite-liberty/pull/3/commits/b16dddabb467fa75ebd7be52388895044c91ecf6
It's not necessary to define recipes in Yaml; without that indirection there's fewer recipes visible to users in the docs, and less confusion about which to run when.

Hope you agree with that change; it's meant as a suggestion, despite the direct commit.

Reply:
> I think Tim is saying to `RevokeSSOCookiesToLogout` to `WebSphereUnavailableSSOCookieMethod`. That way we would no longer need this entry in the Yaml file.

Indeed, applied just now; That just leaves [this comment](https://github.com/openrewrite/rewrite-liberty/pull/3#discussion_r1294744636) before we're good to merge I think.

Reply:
Oh i didnt know we could get rid of the entry in .yml file. I am fine with these changes.